rules/library.bzl

```diff
@@ -961,6 +961,8 @@ def apple_library(name, library_tools = {}, export_private_headers = True, names
     )
 
     if swift_sources:
+        features = kwargs.pop("features", [])
```

Comment:
Should we pull this out at a higher level and forward to "all" places (maybe just those getting `**kwargs` for now)? Since `objc_library` below was getting and could/should also get these `features`.

Reply:
I think the actual Obj-C one (L1067) is getting it via `**kwargs` but you make a good point about the C++ one (L1024).

Reply:
Not if you pop it here, right?

Reply:
Fixed!